x/vm/genesis.go

```diff
@@ -57,6 +57,10 @@ func InitGenesis(
 		}
 	}
 
+	if err := k.AddPreinstalls(ctx, types.DefaultPreinstalls); err != nil {
```

Comment:
Could we add this into the evmd instead? I'm thinking that we could pass in the default preinstalls as a default there, but allow anyone to specify their own precompiles upon genesis inside their app

Reply:
Yeah for sure, I also felt there was something slightly off about this, so I will take another look to make this more reasonable.